src/cheri_types.sail

```diff
@@ -63,6 +63,8 @@
 let reserved_otypes = 4
 let otype_unsealed = -1
 let otype_sentry   = -2 /* Sealed erstwhile or prospective PCC */
+let otype_isentry_load = -3 /* Sealed erstwhile IDC pointing at PCC */
```

Comment:
Is `erstwhile` correct? Isn't this what is going to be `idc` when invoked rather than what was idc when sealed?

Reply:
How embarrassing.